low-level/framework/utils/webservices.py

```diff
@@ -23,7 +23,8 @@
 
 import requests
 from requests.exceptions import Timeout, ConnectionError, HTTPError
-from framework.utils.service_logging import logger
+from framework.base.sspl_constants import SSPL_LOG_PATH
+from cortx.utils.log import Log as logger
```

Comment:
Why here we can't use `from framework.utils.service_logging import logger` ??

Reply:
done.
Note:- we need to initialize logger obj in webservices.py file as well because realstor login api is called even before sspl_ll_d script.